### PR TITLE
Feature/essifi 57

### PR DESCRIPTION
**Pull request by maikel-maas**

### CODE REVIEW

File: src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts [hunk 1]
```diff
@@ -10,20 +13,14 @@ export class PresentationDefinitionWrapperEntity {
...
-  presentation_definition?: PresentationDefinition;
-
   @Column()
   // @ts-ignore
-  id: string;
+  presentation_definition: PresentationDefinition;
 
   @Column()
   // @ts-ignore
-  comment: string;
```
Comment (sksadjad) [inline src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts:22]: why did you remove the comment property?

File: src/entity/presentationWrapper/verifiableCredentialEntity.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import {
```
Comment (sksadjad) [inline src/entity/presentationWrapper/verifiableCredentialEntity.ts:1]: this class is dangerous.. see my comment [here](https://sphereon.atlassian.net/browse/ESSIFI-55?focusedCommentId=34676)

**Review by sksadjad — CHANGES_REQUESTED** [review]
see the remarks about why we need to use wrappers instead of objects when saving in mongodb

File: src/entity/presentationWrapper/verifiableCredentialEntity.ts (hunk 2 above)
Comment (maikel-maas) [reply src/entity/presentationWrapper/verifiableCredentialEntity.ts:1]: I didn't get it. Do you want me to wrap it inside another class? It's already inside the presentation which is wrapped by the presentation wrapper.

File: src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts (hunk 1 above)
Comment (maikel-maas) [reply src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts]: Reverted the changes.

**Review by sksadjad — APPROVED** [review]
looks good
